dspace/config/modules/requestitem.cfg

```diff
@@ -0,0 +1,59 @@
+#---------------------------------------------------------------#
+#----------------REQUEST ITEM CONFIGURATION---------------------#
+#---------------------------------------------------------------#
+
+# Configuration of request-item. Possible values:
+# all - Anonymous users can request an item
+# logged - Login is mandatory to request an item
+# empty/commented out - request-copy not allowed
+#request.item.type = all
+
+# Should all Request Copy emails go to the helpdesk instead of the item submitter?
+#request.item.helpdesk.override = false
+
+# Should a rejection of a copy request send an email back to the requester?
+# Defaults to "true", which means a rejection email is sent back.
+# Setting it to "false" results in a silent rejection.
+#request.item.reject.email = true
+
+# Bundles to use when granting access and inspecting file size, if "all files" is indicated.
+# This supports multiple values. To specify multiple bundles, separate them with a comma
+# or (preferred) repeat the configuration property with additional values.
+# Default: ORIGINAL
+request.item.grant.bundles = ORIGINAL
+#request.item.grant.bundles = PRESERVATION
+
+## By default a granted request will send the file as an attachment.
+# If a secure web link to the bitstream(s) should be sent when files are over a certain file
+# size threshold, enable the following property
+# Default: false
+#request.item.grant.link = true
```

Comment:
My opinion is that this should be the default behavior. A link should be send by default.

Reply:
very happy to change this based on combined reviewer feedback, i just tend to a more conservative approach of not enabling new features by default

Reply:
I agree with @paulo-graca that we may want to default this to `true`.  The reason is that current behavior is *not* ideal because large files cannot always be attached to an email.  So, a default of `true` provides improved behavior as that solves that issue.